core/modules/system/css/system.css

```diff
@@ -148,6 +148,14 @@ div.indentation {
   margin: -1em -0.4em -1em 0.2em;
   padding: 1em 0.6em 1em 0;
 }
+div.form-item-indentation {
+  display: inline-block;
+  width: 1em;
+}
+[dir="rtl"] div.form-item-indentation {
+  display: inline-block;
+  width: 1em;
+}
```

Comment:
This can be removed because it's identical to the LTR version. LTR/RTL versions are only needed when left/right padding/margin/border/etc are affected. A 1em inline-block space will work the same regardless of text direction.

```suggestion
```